### PR TITLE
fix(website): fix different broken icons on the plugin page

### PR DESCRIPTION
**Pull request by christoph-jerolimov**
## Hey, I just made a Pull Request!

Fixes some broken icons on https://backstage.io/plugins/

* `github-codespaces` used the github avatar from https://github.com/github 
* `gitops-cluster`: I couldn't find what kind of icon it was, so I removed it.
* `rollbar`: picked up an icon from their website 
* `tech-radar`: downloaded the material UI icon "track changed" from https://fonts.google.com/icons?icon.set=Material+Icons&selected=Material+Icons+Outlined:track_changes:&icon.query=track+changes&icon.size=80&icon.color=%23e8eaed, it's licensed under Apache. I added manually some license notes to the svg

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/static/img/github.png
Comment (Rugvip) [inline microsite/static/img/github.png:1]: we avoid putting icons in this repo unless we have explicit permission, are there any other sources of these icons that we could use instead?

File: microsite/src/pages/plugins/_pluginCard.tsx [hunk 1]
```diff
@@ -9,14 +9,14 @@ export interface IPluginData {
...
-  iconUrl: string;
+  iconUrl?: string;
   title: string;
   addedDate: string;
   isNew: boolean;
   order?: number;
 }
 
-const defaultIconUrl = 'img/logo-gradient-on-dark.svg';
+const defaultIconUrl = '/img/logo-gradient-on-dark.svg';
```
Comment (Rugvip) [inline microsite/src/pages/plugins/_pluginCard.tsx:19]: might be worth opening a separate PR for this fix as we could ship it straight away :+1:

**Review by Rugvip — COMMENTED** [review]
Thank you!  :tada: 

some concerns around the icons but otherwise good to go

File: microsite/src/pages/plugins/_pluginCard.tsx (hunk 1 above)
Comment (christoph-jerolimov) [reply microsite/src/pages/plugins/_pluginCard.tsx:19]: Sure, give me a sec ;)

Comment (christoph-jerolimov) [reply microsite/src/pages/plugins/_pluginCard.tsx:19]: Hey @Rugvip, see #27920

File: microsite/static/img/github.png
Comment (christoph-jerolimov) [reply microsite/static/img/github.png:1]: Yeah, somehow, I expected that ... I will take a look ... :D

Comment (christoph-jerolimov) [reply microsite/static/img/github.png:1]: I used now remote urls for github and rollbar again. For tech radar I added some license information to the SVG itself. Wdyt?

**Comment by christoph-jerolimov** [comment]
@Rugvip, I updated the icons, can you take a new look? Thank you!

**Comment by christoph-jerolimov** [comment]
Sorry @Rugvip @freben, I know you are all busy, but can you merge this?

**Review by freben — APPROVED** [review]
Alright I think that should be ok
